README.md

```diff
@@ -1,5 +1,9 @@
 # .NET Core Runtime (CoreRT)
-This repo contains the .NET Core runtime optimized for AOT compilation
+This repo contains the .NET Core runtime optimized for ahead of time compilation. To learn more about CoreRT, see the [intro document](Documentation/intro-to-corert.md).
```

Comment:
I'm not sure if you want to handle it in this change, but I'd suggest copying some of the highlights out of this document to here. I remember seeing a few comments on a Hacker News link to CoreRT where people weren't quite sure what CoreRT was or what it was for.